### ISSUE TITLE
Create GraphQL API to get token information

### ISSUE THREAD

**Issue by samuel-uniris** [archethic-foundation/archethic-node#407]
### Is your feature request related to a problem?

Because tokens are created on transaction chain, we might change the supply along the way, causing the addresses to change;
Hence we don't know if the new token is on the right chain.

### Describe the solution you'd like

We can create an API to get the token information as long as the genesis address to provide a better identification of the token.

```
token(address: "0fA02C9....") {
   genesisAddress,
   supply,
   name,
   symbol,
   properties ...
}
```

The API will fetch the genesis and token metadata and send them back in the response

**Comment by apoorv-2204** [comment]
An api also required for getting Oracle Transaction by after time stamp

**Comment by samuel-uniris** [comment]
Closed by #490
